init.go

```diff
@@ -35,6 +36,18 @@ func init() {
 	serverURLF := flag.String("pmm.server-url", "https://127.0.0.1:8443/", "PMM Server URL.")
 	flag.Parse()
 
+	flag.VisitAll(func(i *flag.Flag) {
+		envVar := strings.Replace(strings.ToUpper(i.Name), ".", "_", -1)
+		envVar = strings.Replace(envVar, "-", "_", -1)
+		env, ok := os.LookupEnv(envVar)
+		if ok {
+			err := i.Value.Set(env)
+			if err != nil {
+				logrus.Fatalf("Invalid ENV variable %s: %s", envVar, env)
+			}
+		}
+	})
```

Comment:
The problem with that approach is a lack of documentation – what environment variables should be used?
That code also probably too generic for no good reason.
Maybe it is better to add environment variables names to the flag descriptions, and use `range` over `map[envvar]Flag` instead of visiting all flags.